### ISSUE TITLE
Wire's "About" dialog does not allow copying.

### ISSUE THREAD

**Issue by Siguza** [wireapp/wire-desktop#989]
Oh boy, this is fun. Ran into a bug while reporting another bug.

Wire Version: 3.0.2814
Wire for Web Version: 2017.09.26.1300
OS: macOS Sierra 10.12.6
Do you have an antivirus software installed: Nope

> What steps will reproduce the problem?

1. Open the "About" dialog.
2. Try to copy text from there.
3. Break down in tears because it does not work.

> What is the expected result?

That you can select copy the text displayed there.

> What happens instead?

It won't let you select anything, thus preventing you from copying any text. This is pretty toxic, considering you want this information for your bug reports. And other apps usually allow this... see Finder, VLC Player, VirtualBox, pretty much any app I tried.

**Comment by gregor** [comment]
Thank you for your report raising a valid concern. I will add it to the list of things that we need to address.

**Comment by davidak** [comment]
I also ran into this usability issue.

>What steps will reproduce the problem?

1. Open the "About" dialog.
2. Try to copy text from there.
3. Get crippling depression and consider not to report bugs and end your life instead

I also tried this commands in console:

```
wire-desktop --version
wire-desktop -V
wire-desktop -h
wire-desktop --help
man wire-desktop
```

Also not the best experiance.

---
![screenshot from 2017-10-08 02 08 15](https://user-images.githubusercontent.com/91113/31312786-933838ee-abcd-11e7-915c-d22f7e12feb4.png)
